File: mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# nested for loop to determine composition of each row (will need 4 variables.)
while i in range(1, height + 1):
    
    # left sides spaces
    for j in range(height - i):
        print(" ", end="")
    # left side hashes
    for k in range(i):
        print("#", end="")
    # middle gap
    print("  ", end="")
    # right side hashes
    for l in range(i):
        print("#", end="")
    print()

File: mario.c
#include <cs50.h>
#include <stdio.h>

//start command
int main (void)

{
//declare variable so it doesn't flip out later
    int n; //only use "int" once, when you're declaring the variable? Created an error earlier when i used it below this
    
//do while loop (input) this will determine number of rows
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n >8);
    
//nested for loop to determine composition of each row (will need 4 variables.)
    for (int i = 0; i < n; i++)
    {
        
        //left sides spaces
        for (int j = 1; j < (n-i); j++)
        {
            printf(" ");
        }
        
        //left side hashes
        for (int k = 0; k <= i; k++)
        {
                printf("#");
        }
        
        //middle gap
        printf("  ");
        
        //right side hashes
        for (int l = 0; l <= i; l++)
        {
            printf("#");
        }
        
        printf("\n");
    }
}
